### PR TITLE
Bug/login page layout

### PR DESCRIPTION
**Pull request by jimjamesjimathy**
I implemented flexbox to pull together the login component and the text box. I then added new colors to the font so that it is easier to read. I also added media queries so that the text won't fold over upon resizing. And lastly, I added the blue border to the login component when the screen is at its smallest so that it doesn't disappear into the background.

Fixes # ( unreadable text and resizing issues )

## Loom Video

 - https://www.loom.com/share/72b59cd022dc49efbcfc3aa0e35e0b96

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have removed unnecessary comments/console logs from my code
- [x] I have made corresponding changes to the documentation if necessary (optional)
- [x] My changes generate no new warnings
- [x] I have checked my code and corrected any misspellings
- [x] No duplicate code left within changed files
- [x] Size of pull request kept to a minimum
- [x] Pull request description clearly describes changes made & motivations for said changes

### CODE REVIEW

File: src/components/pages/LandingPrograms/CoderSitters.js [hunk 1]
```diff
@@ -8,50 +8,52 @@ export default function CoderSitters() {
...
+            <Title className="pl__na__sitter__title" level={2}>
+              CODERSITTERS
             </Title>
-
-            {/* INFO PARAGRAPH */}
-            <Text className="pl__na__sitter__text__container">
-              Our lessons include playing age-appropriate
-              <br />
-              coding games online, in tandem with a wide
+            {/* CODERSITTERS INTRO PARAGRAPH */}
```
Comment (WRWPhillips) [inline src/components/pages/LandingPrograms/CoderSitters.js:17]: You made this whole section look a lot more intuitive from the code side as well.

File: src/components/pages/LandingPrograms/ExplorePrograms.js [hunk 2]
```diff
@@ -33,15 +32,13 @@ export default function ExplorePrograms() {
...
-              <img
-                className="pl__na__top__right__light__bulbs"
-                src={lightBulbs} // located inside styles/ProgramsLandingStyles/assets
-                alt="peach star"
-              />
-            </div>
+          <div className="pl__na__top__right__image__container">
+            {/* BLOCKER:If anyone has experience with rendering images, please take on this part of the project.*/}
+            <img
+              className="pl__na__top__right__light__bulbs"
```
Comment (WRWPhillips) [inline src/components/pages/LandingPrograms/ExplorePrograms.js:38]: Who wrote these classnames? They are so bizarre.

File: src/styles/LandingPageStyles/index.less [hunk 3]
```diff
@@ -320,6 +316,28 @@
   }
 }
 
+//** Landing page styles - Tablet fix **//
```
Comment (WRWPhillips) [inline src/styles/LandingPageStyles/index.less:319]: Really meticulous styling, target by target and line by line pretty much. The format now shows a lot of work was put into it! Nice work on responsiveness!

File: src/styles/NavbarStyles/index.less [hunk 4]
```diff
@@ -2,64 +2,53 @@
...
     display: flex;
-    width: 40%;
-    justify-content: space-between;
-    margin-left: 0.6rem;
+    align-items: center;
+    justify-content: space-evenly;
+    width: 65%;
 
     .navbar__navLink {
+      height: 55px;
```
Comment (WRWPhillips) [inline src/styles/NavbarStyles/index.less:29]: Changes on navLink were super super needed

**Review by WRWPhillips — APPROVED** [review]
Overall a ton of changes, but the code is clean and semantic and the new layour and responsiveness of design speaks for itself really.

**Review by rjgeng — APPROVED** [review]
Myriad of changing, but very clear and neat coding. great job!

**Pull request merged**